components/log-viewer-webui/client/webpack.config.js

```diff
@@ -80,16 +99,11 @@ const config = {
     resolve: {
         extensions: [
             ".js",
-            ".jsx",
             ".json",
+            ".ts",
+            ".tsx",
         ],
     },
 };
 
-export default () => {
```

Comment:
@kirkrodrigues I don't recall if we discussed this for #468 - Did we have to export a function and change the `config.mode` dynamically?  Can we simply define the `mode` in the above `config` object and return the object directly?

Reply:
Yeah, that's my understanding based on this: https://webpack.js.org/configuration/configuration-types/#exporting-a-function

I think the old code was wrong since it should've been using the `env` argument to determine whether it was prod/dev rather than `isProduction`. Based on a glance at [this](https://webpack.js.org/guides/production/#specify-the-mode), I guess we shouldn't be using process.env.NODE_ENV in a webpack config.

Reply:
Gotcha. Let me defer this after investigating the viability of turning the app into a Vite-React app.

Reply:
The attempt to migrate to Vite was effortless and they provide all what we need with very simple configuration interfaces. I think we can stick with Vite in this project.